rosco-web/config/rosco.yml

```diff
@@ -202,25 +202,25 @@ openstack:
   bakeryDefaults:
     identityEndpoint: ${OS_AUTH_URL:}
     domainName: Default
-    floatingIpPool: ext_vlan1472_net
+    floatingIpPool: ext_network
```

Comment:
Why? The ones that currently use environment variables are tied to the environment variables that are provided via the openstack rc script that can be downloaded from the web console.

afaik you can do this in Spring boot for environment variables:

```
OPENSTACK_BAKERYDEFAULTS_FLOATINGIPPOOL=ext_network
```